src/services/completions.ts

```diff
@@ -4850,6 +4850,12 @@ function tryGetObjectTypeDeclarationCompletionContainer(sourceFile: SourceFile,
         return findAncestor(contextToken, isClassLike) as ObjectTypeDeclaration;
     }
 
+    // class C { prop = ""\n | }
+    if (contextToken && isPropertyDeclaration(contextToken.parent) && isClassLike(location)
+        && getLineAndCharacterOfPosition(sourceFile, contextToken.getEnd()).line !== getLineAndCharacterOfPosition(sourceFile, position).line) {
+        return location;
+    }

```

Comment:
So I was going to ask about a test case like

```ts
interface Interface {
    method(): void;
}

export class Class implements Interface {
    property = "yadda" + "yadda"
    
    /*$*/
}
```

and I have *no idea* why this example has reasonable completions, whereas the code in the original issue doesn't. Do you have an idea right now?

Reply:
Is it from the `if (!isFromObjectTypeDeclaration(contextToken)) {` code below? Maybe we should just get rid of that check?